cpp/src/io/csv/reader_impl.cu

```diff
@@ -116,10 +116,10 @@ std::vector<std::string> setColumnNames(std::vector<char> const &header,
                                         int header_row,
                                         std::string prefix)
 {
-  std::vector<std::string> col_names;
+  std::vector<std::string> col_names_;
```

Comment:
This one is not a private data member but just a regular stack variable. Seems this would cause confusion with the private member.

Reply:
Oops. This mistake is just another reason we should rename them. Thanks! :)